nsot/api/serializers.py

```diff
@@ -296,3 +298,37 @@ def validate(self, attrs):
         else:
             msg = 'Must include "email" and "secret_key"'
             raise exc.ValidationError(msg)
+
+
+#######
+# Hooks
+#######
+
+class HookSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Hook
+
+    def to_representation(self, obj):
+        """Explicitly ensure Hook serializes with all its fields"""
+        rep = model_to_dict(obj)  # This doesn't include 'auto_now' fields
+        rep['created'] = str(obj.created)
+        rep['updated'] = str(obj.updated)
+
+        return rep
+
+
+class HookCreateUpdateSerializer(HookSerializer):
+    class Meta:
+        model = Hook
+        fields = ('event', 'target', 'global_hook')
+
+    def create(self, validated_data):
+        """Explicitly include the logged-in user"""
+        ModelClass = self.Meta.model
+        validated_data['user'] = self.context['request'].user
+        try:
+            obj = ModelClass.objects.create(**validated_data)
```

Comment:
I would prefer that we still call the parent's `.create()` (e.g.`obj = super(HookSerializer, self).create(validated_data)`) here, because it's doing more than just calling the model's `.create()` method. Everything else is great!